mw-webapp/src/service/DayReportService.ts

```diff
@@ -35,7 +35,7 @@ export class DayReportService {
     const dayReportRaw = await getDoc(doc(db, PATH_TO_DAY_REPORTS_COLLECTION, uuid));
     const dayReportDTO = documentSnapshotToDTOConverter<DayReportDTO>(dayReportRaw);
 
-    const validatedDayReportDTO = DayReportDTOSchema.parse(dayReportDTO);
+    const validatedDayReportDTO = DayReportDTOSchema.parse({...dayReportDTO, s: 1});
```

Comment:
please explain what does s: 1 mean?

Reply:
So you can see error

Reply:
yeah, thanks for this, good idea! because of this, we was able to check behavior in preview!